src/v2/controllers/FormController.js

```diff
@@ -165,6 +165,12 @@ export default Controller.extend({
       return;
     }
 
+    const identifier = model.get('identifier');
+    const transformUsername = this.settings.get('features.transformUsername');
+    if (_.isFunction(transformUsername) && identifier) {
```

Comment:
what if the identifier is an email ? We should check with backend on this..

Reply:
It's up to the dev what they want to do with transformUsername. 
Here's an example from SIW readme:
```
// The callback function is passed two arguments:
// 1) username: The name entered by the user
// 2) operation: The type of operation the user is trying to perform:
//      - PRIMARY_AUTH
//      - FORGOT_PASSWORD
//      - UNLOCK_ACCOUNT
transformUsername: function (username, operation) {
  // This example will append the '@acme.com' domain if the user has
  // not entered it
  return username.indexOf('@acme.com') > -1
    ? username
    : username + '@acme.com';
}
```